Geometry/TrackerCommonData/plugins/dd4hep/DDCutTubsFromPoints.cc

```diff
@@ -181,7 +182,7 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
 
   for (unsigned i = 1; i < segments.size() - 1; i++) {
     solid = dd4hep::UnionSolid(
-        solidOutput + "_uni" + std::to_string(i + 1), solid, segments[i], dd4hep::Position(0., 0., offsets[i] - shift));
+        solidOutput + "_uni_" + std::to_string(i + 1), solid, segments[i], dd4hep::Position(0., 0., offsets[i] - shift));
```

Comment:
The corresponding line in the old DD version is:
```
DDName unionname(solidOutput.name() + "_uni_" + std::to_string(i + 1), solidOutput.ns());
```
Adding the `_` allows the names to match exactly between old DD and DD4hep.